Mathlib/RingTheory/Localization/Away/Basic.lean

```diff
@@ -358,6 +358,33 @@ instance away_snd {R S} [CommSemiring R] [CommSemiring S] :
 
 end Prod
 
+open Pointwise in
+/-- Suppose `J ≤ I` are ideals of `R`, then `R / I` is the localization away from `r : R / J`
+if `r - 1 ∈ I` and for some `n`, `r ^ n • I ≤ J`.
+For sake of usability, we state this for surjective ring maps instead of ideals. -/
+lemma Away.of_sub_one_mem_ker {R S T : Type*}
+    [CommRing R] [CommRing S] [CommRing T] [Algebra R S] [Algebra R T]
+    [Algebra S T] [IsScalarTower R S T]
+    (h₁ : Function.Surjective (algebraMap S T))
+    (h₂ : Function.Surjective (algebraMap R S))
+    (r : R) (hr : r - 1 ∈ RingHom.ker (algebraMap R T))
```

Comment:
Is this form more useful than `(hr : algebraMap R T r = 1)`?

Reply:
It still won't be defeq and be a simp away for `T = R / I` right? In that case should you add a `Ideal.Quotient.mk I r = 1 <-> r - 1 \in I` lemma and use that?